released/packages/coq-elpi/coq-elpi.1.9.6/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+name: "coq-elpi"
+version: "dev"
```

Comment:
?

Reply:
thanks, I'll let CI run fully and then fix this one